core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { OperatorLabelComponent } from './operator-label.component';
+import { OperatorMetadataService } from '../../../service/operator-metadata/operator-metadata.service';
+import { StubOperatorMetadataService } from '../../../service/operator-metadata/stub-operator-metadata.service';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+
+describe('OperatorLabelComponent', () => {
+  // let component: OperatorLabelComponent;
+  // let fixture: ComponentFixture<OperatorLabelComponent>;
+
+  // beforeEach(async(() => {
+  //   TestBed.configureTestingModule({
+  //     declarations: [ OperatorLabelComponent ],
+  //     providers: [
+  //       { provide: OperatorMetadataService, useClass: StubOperatorMetadataService }
+  //     ],
+  //     schemas: [ NO_ERRORS_SCHEMA ]
```

Comment:
This file is tool generated. Then, why is this portion commented out?

Reply:
the test case will be added back into a future PR